design/ingressroute-design.md

```diff
@@ -353,23 +353,19 @@ _TBD_
 
 Metrics are essential to any system. Contour will expose a `/metrics` Prometheus endpoint with the following metrics:
 
-- **contour_ingressroute_total (gauge):** Number of IngressRoutes
+- **contour_ingressroute_total (gauge):** Total number of IngressRoutes objects that exist regardless of status (i.e. Valid / Invalid / Orphaned, etc). This metric should match the sum of `Orphaned` + `Valid` + `Invalid` IngressRoutes.
   - namespace
   - vhost
```

Comment:
I won't have a `vhost` for any orphaned routes. @rosskukulinski I think we need to take off the vhost for this metric, or remove orphaned from it.

Reply:
Are Root ingressroutes in namespaces that are not allowed to make roots classified as Orphaned?  We SHOULD have a metric that tracks that, but orphaned is probably not the right one.

+1 for removing vhost for orphaned